### PR TITLE
Add TypeScript declarations to iron-menu-behavior.

### PR DESCRIPTION
**Pull request by aomarks**
This PR adds TypeScript declarations generated by https://github.com/Polymer/gen-typescript-declarations/

These declarations can be re-generated by running `npm run update-types`.

Tracker: https://github.com/Polymer/gen-typescript-declarations/issues/79

### CODE REVIEW

File: iron-menu-behavior.d.ts [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+     */
+    attrForItemTitle: string|null|undefined;
+    disabled: boolean|null|undefined;
+
+    /**
+     *  https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState
+     */
+    _MODIFIER_KEYS: any[]|null;
+    _SEARCH_RESET_TIMEOUT_MS: number;
+    _previousTabIndex: number;
```
Comment (TimvdLippe) [inline iron-menu-behavior.d.ts]: I think we should make all these 3 private.

File: iron-menu-behavior.d.ts [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+     */
+    readonly focusedItem: object|null;
+
+    /**
+     * The attribute to use on menu items to look up the item title. Typing the first
+     * letter of an item when the menu is open focuses that item. If unset, `textContent`
+     * will be used.
+     */
+    attrForItemTitle: string|null|undefined;
+    disabled: boolean|null|undefined;
```
Comment (TimvdLippe) [inline iron-menu-behavior.d.ts]: Default value is `false`, so I think this should be just `boolean`

**Review by TimvdLippe — APPROVED** [review]
2 small nits, other than that LGTM

File: iron-menu-behavior.d.ts (hunk 2 above)
Comment (aomarks) [reply iron-menu-behavior.d.ts]: Done.

File: iron-menu-behavior.d.ts (hunk 1 above)
Comment (aomarks) [reply iron-menu-behavior.d.ts]: Done.

**Pull request merged**